easse/resources/data/test_sets/turkcorpus/tune.truecase.detok.simp.7

```diff
@@ -0,0 +1,2000 @@
+Adjacent counties are Marin (to the south), Mendocino (to the north), Lake (northeast), Napa (to the east), and Solano and Contra Costa (to the southeast) .
```

Comment:
It's odd because I used the same script in all files. I've run it again but it still doesn't work. The most noticeable issue is the final periods in each sentence. We could fix that manually (time-consuming) or just leave it like that (so it's in the "same" conditions as the other that were not manually fixed). Opinions?

Reply:
Fixed it. It had to do with the original codification of the file. It's all good now.